src/components/CMessageTable.vue

```diff
@@ -47,9 +47,8 @@
           ></c-message-row>
           <tr :key="m.messageId + 0" v-if="openedRows.includes(m.messageId)">
             <td class="border text-xs px-4 py-2" :colspan="columns.length">
-              <p class="text-xs whitespace-normal ml-2 bg-white">
-                Queue Name: {{ queueName(m.actorName) }}
-              </p>
+              <p class="text-xs">Message Id: {{ m.messageId }}</p>
```

Comment:
could you align args, kwargs message Id and queue name ?

![image](https://user-images.githubusercontent.com/6851979/84630134-c7272380-aeeb-11ea-8641-5502cdc16e43.png)

And write the title of the fields in bold ?